### DISCUSSION TITLE
Adding an option to have 2 heaters turning on at the same time on 1 thermostat

### DISCUSSION THREAD

**Discussion by HoM3r17** [jmcollin78/versatile_thermostat#501] [Ideas]
Hi,

First, thanks for this integration, Thermostat was the only thing I left on Jeedom until now, because their thermostat implementation is much more advanced than what's integrated to HA. Thanks to you, this is no more the case !

The only "complaint" I've got is the handling of the heating with multiple heating devices on the same thermostat.

I've read the "readme" and it's "Only the first radiator heats", and it's really nice to thing about the consumption and peak power, but for exemple in my case I've got a 3 phase installation, and these heaters are on different phases, and installed in a verry big room, witch takes a lot of time to heat up, so I don't really care on the peak power, but would prefer it to hit the target temperature faster.

Would it be possible to add a checkbox to enable / disable this feature when multiple heaters are used ?

Thanks for reading !

Regards.

**Comment by jmcollin78** [comment]
Hello,

I'm not sure to understand what you want exactly. When multiple heaters are used (that is my case and also with 3 phases installation), all heaters are used sequentially or in parallel if more power is needed. What I want is to distribute the power as much as possible on the n heaters instead of only one heats. That is exactly what does VTherm.
